.github/workflows/publish.yml

```diff
@@ -1,7 +1,7 @@
 on:
   push:
     tags:
-      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+      - 'v[0-9]+.[0-9]+.[0-9]*' # Push events on tags matching regular expression, i.e. v20.15.10
```

Comment:
Use "\." in place of "." to constrain regex to dot. Right now the regex matches i.e. 'v1a2b'

Reply:
reverted it